includes/abstracts/class-llms-rest-users-controller.php

```diff
@@ -667,7 +669,10 @@ protected function update_additional_data( $object_id, $prepared, $request ) {
 
 		if ( ! empty( $request['roles'] ) ) {
 			$user = $object->get_user();
-			array_map( array( $user, 'add_role' ), $request['roles'] );
+			$user->set_role('');
```

Comment:
ops @pondermatic missed this. It should be 
```suggestion
			$user->set_role( '' );
```

That's why automatic tests fail here.